src/main/java/com/craftaro/ultimateclaims/dynmap/DynmapManager.java

```diff
@@ -56,8 +56,8 @@ public void refresh() {
 
         if (this.plugin.getClaimManager() != null) {
             for (Claim claim : this.plugin.getClaimManager().getRegisteredClaims()) {
-                if (Bukkit.getWorld(claim.getFirstClaimedChunk().getWorld()) == null) {
-                    continue;
+                if (claim.getFirstClaimedChunk() != null && Bukkit.getWorld(claim.getFirstClaimedChunk().getWorld()) == null) {
```

Comment:
Just a little nitpick (so just a comment not a request for change) but it might make sense to store the result of `Claim#getFirstClaimedChunk()`.
In theory both calls to it should return the exact same result - But you can't be sure in some/most Songoda plugins because of Multi-Threading abuse ,_,